spec/shopping_cart_spec.rb

```diff
@@ -0,0 +1,6 @@
+require './lib/models/shopping_cart.rb'
+
+describe Shopping_cart do
+  xit { is_expected.to have_property :orders }
```

Comment:
The test needs to be rewritten. Need more input on how to write this then we are using sessions to store data.